source/ControllableHero.hx

```diff
@@ -0,0 +1,32 @@
+import flixel.FlxG;
+import flixel.FlxSprite;
+import flixel.util.FlxColor;
+
+class ControllableHero extends FlxSprite
+{
+	public function new(x:Int, y:Int)
+	{
+		super(x, y);
+		this.makeGraphic(50, 50, FlxColor.WHITE);
+	}
+
+	override function update(elapsed:Float)
+	{
+		if (FlxG.keys.pressed.LEFT)
+		{
+			this.x--;
```

Comment:
This applies across to how movement's done in both classes for this PR -

This way of moving works (and nice work making it functional, I know this has involved diving into some unfamiliar territory) but it'll have that flaw we talked about in one of our previous lectures where the game logic is tightly coupled to the rate that visual frames are drawn to the screen. 

If someone runs it at 30fps, the objects move half as fast. If someone wanted to play at 240fps, the game's probably unplayable, assuming everything's coupled to framerate like this movement is. 

We'll be talking about how HaxeFlixel helps us dodge this problem with `FlxObject.velocity` during our next class. No need to change anything for this PR, I just wanted to bring it to your attention now since it's featured pretty heavily in your solution.